### PR TITLE
DCK-1561: Switch to use the Dock API instead of the SDK

### PR DESCRIPTION
**Pull request by mike-parkhill**

### CODE REVIEW

File: components/verifier-modal.js [hunk 1]
```diff
@@ -1,77 +1,34 @@
...
   Button,
 } from '@material-ui/core';
 
-// Import Dock SDK utils
-import { DockResolver, MultiResolver, UniversalResolver } from '@docknetwork/sdk/resolver';
-import { verifyCredential, verifyPresentation } from '@docknetwork/sdk/utils/vc';
-import dock from '@docknetwork/sdk';
-
-// Create the resolver
-const universalResolverUrl = 'https://uniresolver.io';
```
Comment (SamHellawell) [inline components/verifier-modal.js:24]: the dock api currently only verifies credentials that use dock DIDs. we need to add this universal resolver code to the API too

File: components/verifier-modal.js [hunk 2]
```diff
@@ -1,77 +1,34 @@
...
-    verifyResult = await verifyCredential(json, verifyParams);
-  }
+  const verifyResult = await axios.post(`api/verify/`, json);
 
   if (!verifyResult) {
     throw new Error('No presentation or credential provided');
   }
 
-  if (verifyResult.verified) {
+  if (verifyResult.data.verified) {
```
Comment (SamHellawell) [inline components/verifier-modal.js:23]: `verifyResult.data` is repeated often, could use destructuring

File: pages/api/verify/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const axiosHeaders = {
+    headers: {
```
Comment (SamHellawell) [inline pages/api/verify/index.js:4]: wrong tabulation in this file

File: components/verifier-modal.js (hunk 2 above)
Comment (mike-parkhill) [reply components/verifier-modal.js]: fixed

File: pages/api/verify/index.js (hunk 3 above)
Comment (mike-parkhill) [reply pages/api/verify/index.js]: fixed

File: components/verifier-modal.js (hunk 1 above)
Comment (SamHellawell) [reply components/verifier-modal.js:24]: we can accept the PR even before this change aslong as it happens right after

**Review by SamHellawell — APPROVED** [review]

File: components/verifier-modal.js (hunk 1 above)
Comment (mike-parkhill) [reply components/verifier-modal.js:24]: working on the API now

**Pull request merged**